src/blueprints/tv2/factories/tv2-viz-graphics-timeline-object-factory.ts

```diff
@@ -110,5 +114,50 @@ export class Tv2VizGraphicsTimelineObjectFactory implements Tv2GraphicsTimelineO
     }
   }
 
+  public createFullGraphicsTimelineObject(blueprintConfiguration: Tv2BlueprintConfiguration, manifest: Tv2GraphicActionManifest): TimelineObject {
+    return {
+      id: '',
+      enable: {
+        start: 0
+      },
+      priority: 1,
+      layer: this.getLayerNameFromGraphicTarget(Tv2GraphicsTarget.FULL),
+      content: this.createFullGraphicTimelineObjectContent(blueprintConfiguration, manifest),
+      classes: ['full']
```

Comment:
`classes` are used to reference stuff when building the `Timeline`. Our implementation of the `Timeline` doesn't use `classes`, so we should just ignore them (meaning don't include them in our Blueprints)

Reply:
Removed.